Client/Frontend/Browser/Playlist/Controllers/PlaylistCarplayController.swift

```diff
@@ -780,7 +780,7 @@ extension PlaylistCarplayController {
   }
 
   func load(url: URL, autoPlayEnabled: Bool) -> AnyPublisher<Void, Error> {
-    load(asset: AVURLAsset(url: url), autoPlayEnabled: autoPlayEnabled)
+    load(asset: AVURLAsset(url: url, options: AVAsset.defaultOptions), autoPlayEnabled: autoPlayEnabled)
```

Comment:
Small style nit: Can't `AVAsset` prefix be omitted in all these references? (i.e. `AVAsset.defaultOptions` → `.defaultOptions`)

Reply:
No because the `options` parameter is `[String: Any]?` but the extension is on `AVAsset` so when I do:
`AVURLAsset(url: "....", options: .defaultOptions)` the compiler sees: `Dictionary<String, Any>.defaultOptions` which doesn't exist since it exists only on `AVAsset`.

Reply:
Thanks, didn't realize that!